order.go

```diff
@@ -139,20 +139,30 @@ type OrderList struct {
 
 // OrderListParams is the set of parameters that can be used when listing orders.
 type OrderListParams struct {
-	ListParams   `form:"*"`
-	Created      *int64            `form:"created"`
-	CreatedRange *RangeQueryParams `form:"created"`
-	Customer     *string           `form:"customer"`
-	IDs          []*string         `form:"ids"`
-	Status       *string           `form:"status"`
+	ListParams        `form:"*"`
+	Created           *int64                         `form:"created"`
+	CreatedRange      *RangeQueryParams              `form:"created"`
+	Customer          *string                        `form:"customer"`
+	IDs               []*string                      `form:"ids"`
+	Status            *string                        `form:"status"`
+	StatusTransitions *StatusTransitionsFilterParams `form:"status_transitions"`
+}
+
+// StatusTransitionsFilterParams are parameters that can used to filter on status_transition when listing orders.
+type StatusTransitionsFilterParams struct {
+	StatusTransitions `form:"*"`
+	CanceledRange     *RangeQueryParams `form:"canceled"`
```

Comment:
Could you also add a field for each of `Canceled`, `Fulfilled`, `Paid`, and `Returned` which is a `*int64`? This is how we generally support either a range or a basic Unix timestamp like for `Created`. From the docs:

> A filter on the list based on the object canceled field. The value can be a string with an integer Unix timestamp, or it can be a dictionary with the following options:

Reply:
I had implicitly provided these fields by embedding `StatusTransitions` in `StatusTransitionsFilterParams`.
Should I instead explicitly include each of the fields in `StatusTransitionsFilterParams`?

Reply:
Thanks. Done.